### PR TITLE
Issue 38

### LINKED ISSUE #38: Documentation

### PR DESCRIPTION
**Pull request by pinclau**

### CODE REVIEW

File: mkdocs.yml
Comment (alexmerlin) [inline mkdocs.yml:1]: Replace all `v5` occurences with `v3`.

**Review by alexmerlin — CHANGES_REQUESTED** [review]
- also add SECURITY.md file - example [here](https://github.com/dotkernel/dot-annotated-services/blob/5.0/SECURITY.md) - don't forget to adapt it to your package
- do you plan to add more files under `events` and `plugins` directories? if no, move their contents up one level

File: mkdocs.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  - v3:
+      - Overview: v5/overview.md
+      - Installation: v5/installation.md
+      - Configuration: v5/configuration.md
+      - Usage: v5/usage.md
+      - Plugins: v5/plugins/plugins.md
+      - Events: v5/events/events.md
+site_name: dot-controller
+site_description: "DotKernel's controller package"
+repo_url: "https://github.com/dotkernel/dot-controller"
```
Comment (alexmerlin) [inline mkdocs.yml:19]:
```suggestion
repo_url: "https://github.com/dotkernel/dot-controller"
plugins:
  - search
```

File: composer.json [hunk 2]
```diff
@@ -55,7 +55,8 @@
     "config": {
         "sort-packages": true,
         "allow-plugins": {
-            "dealerdirect/phpcodesniffer-composer-installer": true
+            "dealerdirect/phpcodesniffer-composer-installer": true,
+            "laminas/laminas-dependency-plugin": true
```
Comment (alexmerlin) [inline composer.json]:
```suggestion
            "laminas/laminas-dependency-plugin": false
```

**Review by alexmerlin — CHANGES_REQUESTED** [review]

**Review by alexmerlin — APPROVED** [review]

**Review by arhimede — APPROVED** [review]

**Pull request merged**